its/plugin/src/test/java/org/sonarsource/slang/ExternalReportTest.java

```diff
@@ -110,4 +155,12 @@ public void rubocop() {
       .collect(Collectors.toList());
   }
 
+  private String replaceInReport(Path sourceReportPath, String placeHolder, Path target) throws IOException {
```

Comment:
The signature of this method is not very clear. Returning a `Path` may clarify it. Finding a better name would also help.

Reply:
I will rename it: createTemporaryReportFileFromTemplate